core/app/components/FormBuilder/types.ts

```diff
@@ -27,7 +27,7 @@ export type InputElement = baseElement & {
 	type: ElementType.pubfield;
 	fieldId: PubFieldsId;
 	required: boolean | null;
-	label?: string | null;
+	label: never;
```

Comment:
is `label` used at all anymore now that we have `config`? should we remove it as part of the migration?

Reply:
i had initially removed it, but then i realized the buttons do use the label. i could make buttons use config instead and then we could drop the label. the unique constraint preventing duplicate button names would need some reworking too

Reply:
ah right, I forgot about the buttons! hmm if it makes the unique constraint not work then I think we can keep it as-is (though maybe with a comment that it's only used for labels?)

Reply:
will do! i also updated the explicit types in the FormBuilder/types.ts file so that label is `never` on input elements

Reply:
arguably we could make the unique constraint work (just put it on the config field), but it would also mean that buttons also have to be subject to all the annoying caveats of storing config in a json file (weak types/validation required, risky updates). so i'm going to leave it as is for now because of that mainly!